### DISCUSSION TITLE
predictions_tab

### DISCUSSION THREAD

**Discussion by wgova** [oegedijk/explainerdashboard#165] [Ideas]
One of the challenges, when you POC, and are making a case for your model, is that you would like to make predictions for a few rounds before deploying. It would be cool to add a tab to make some predictions and test the model as well

**Comment by oegedijk** [comment]
Hi @wgova, currently the dashboard typically investigates test data, so it already kind of does that. But how would you invision this? Ability to manually input data?

**Reply by wgova** [reply]
I am not 100% sure, but when you deploy this dashboard, it won't be able to take in new, "unseen" data and make predictions. Perhaps I will give it a shot and showcase it. I was suggesting an end-to-end ML dashboard that not only trains and tests, but also makes predictions on new data like this [Streamlit Prophet](https://share.streamlit.io/maximelutel/streamlit_prophet/main/streamlit_prophet/app/dashboard.py)

For more users to collaboratively play with the model and give feedback using their own datasets. This is usually the case with smaller teams, that need to build more infrastructure to scale their models. They can already get value out of the models, and perform batch predictions, and download the results to make decisions while scaling and infrastructure is being set up or evaluated
